src/System.Private.CoreLib/src/System/Environment.UWP.cs

```diff
@@ -46,5 +46,11 @@ public static string MachineName
                 throw new PlatformNotSupportedException();
             }
         }
+
+        public static void Exit(int exitCode)
+        {
+            // Store apps have their lifetime managed by the PLM
+            throw new PlatformNotSupportedException();
```

Comment:
There is [Application.Exit](https://msdn.microsoft.com/en-us/library/windows/apps/windows.ui.xaml.application.exit.aspx) but that work needs to happen on the Project N side. I can open a bug when this is merged...